content/help/ops/telemetry/missing-metrics/index.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Missing Metrics
+description:
+weight: 10
+---
+
+The procedures below help you diagnose problems where metrics
+you are expecting to see reported and not being collected.
+
+The expected flow for metrics is:
+
+1. Envoy reports attributes from requests asynchronously to Mixer in a batch.
+
+1. Mixer translates the attributes into instances based on the operator-provided configuration.
+
+1. Mixer hands the instances to Mixer adapters for processing and backend storage.
+
+1. The backend storage systems record the metrics data.
+
+The Mixer default installations include a Prometheus adapter and the configuration to generate a [default set of metric values](https://istio.io/docs/reference/config/policy-and-telemetry/metrics/) and send them to the Prometheus adapter. The Prometheus adapter configuration enables a Prometheus instance to scrape Mixer for metrics.
+
+If the Istio Dashboard or the Prometheus queries don’t show the expected metrics, any step of the flow above may present an issue. The following sections provide instructions to troubleshoot each step.
+
+## Verify the Mixer post-install job completed successfully
```

Comment:
This section (`Verify the Mixer post-install job completed successfully`) can be removed entirely. That job was removed from helm some time within the last week.